src/components/hooks/useMeetingControl.js

```diff
@@ -45,6 +46,10 @@ export default function useMeetingControl(type, meetingID) {
   }, [meetingID, control]);
 
   return control
-    ? [(value) => control.action(meetingID, value), display]
+    ? [(value) => control.action({
```

Comment:
I've changed the argument for `action()` to an object now as this will be scalable for additions in the future. The `action()` is common for all the various types of controls on the widget.